go-controller/pkg/util/node_annotations.go

```diff
@@ -235,9 +235,15 @@ func ParseNodeManagementPortMACAddress(node *kapi.Node) (net.HardwareAddr, error
 	return net.ParseMAC(macAddress)
 }
 
-func SetNodePrimaryIP(nodeAnnotator kube.Annotator, protocol iptables.Protocol, nodeIP string) error {
-	ipFamily := getFamilyFromProtocol(protocol)
-	return nodeAnnotator.Set(fmt.Sprintf("%s-%s", ovnNodeCIDR, ipFamily), nodeIP)
+func SetNodePrimaryIP(nodeAnnotator kube.Annotator, nodeIPNetv4, nodeIPNetv6 *net.IPNet) error {
```

Comment:
This should be moved to the previous commit where you first added `SetNodePrimaryIP`